### ISSUE TITLE
--no-uploads flag does not work

### ISSUE THREAD

**Issue by kstefanini** [welaika/wordmove#422]
Hello,

Just figure out that `--no-uploads` flag doesn't work, uploads are pulled anyway despite `--no-uploads` flag :
```
$ wordmove pull -e preprod --all --no-uploads
...
▬▬ ✓ Pulling Uploads ▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬
...
receiving incremental file list
./
2016/
...
group-avatars/99/

sent 1,644 bytes  received 118,504 bytes  34,328.00 bytes/sec
total size is 577,056,794  speedup is 4,802.88
```

What i have miss ?

I'm running Wordmove on Debian Jessy/sid,
ruby 2.4.1p111 (2017-03-22 revision 58053) [x86_64-linux],
Wordmove 2.1.3,
rvm 1.29.3

PS : thanks for this amazing work, Wordmove is a great and handy tool !

**Comment by Spone** [comment]
Same here.

**Comment by pioneerskies** [comment]
@kstefanini @Spone the issue was automatically closed by my commit; never intended to stop the conversation :)

I've published a fix in version 2.1.4, so I'd ask you to go and test it if you like.

Backstory:

1. Yes it was a bug, so @kstefanini did not missed anything
2. we use a command line builder 3rd party lib and it automatically creates `--no-` options in help screen for those options declared as booleans
3. actually no one ever implemented the logic to handle those automatically created *reverse* options when the `--all` flag is passed to the program

In the commit the is the fix together with a simple test which should guarantee no future regressions.

Thanks for contributing with your reports guys :) Really appreciated
